wiki/datasets/open-source-datasets.md

```diff
@@ -0,0 +1,130 @@
+This is an article to teach you how to make your own dataset or where to find open-source datasets that are free to use and download.
+
+## Creating a Custom Dataset
+Capture your own images with a camera then create labels for each image that indicates the bounding boxes and IDs of the object class captured.
+
+*Option 1:*
+Create labels for all of the images using Yolo_mark [1]. The repo and instructions for use can be found here: <https://github.com/AlexeyAB/Yolo_mark>. These labels will be made in the darknet format. 
```

Comment:
All links need to be in Markdown format. https://guides.github.com/features/mastering-markdown/